mobile/src/lib/cozy-helper.js

```diff
@@ -34,3 +34,16 @@ export const initBar = () => {
     lang: 'en'
   })
 }
+
+export const isClientRegistered = async (client) => {
+  return await cozy.client.auth.getClient(client).then(client => true).catch(() => false)
```

Comment:
In the catch, you can use the error. I think the message should be `Client has been revoked` if the client has been revoked. If it's not the case, you can check the status code. If you have a status code (probably 401), you are online. If the status code is missing, you can suppose you are offline.